test/unit/language-extensions/table.spec.ts

```diff
@@ -332,6 +332,16 @@ describe("LuaTable extension interface", () => {
             .expectToEqual({ baz: 5 });
     });
 
+    test("table add", () => {
```

Comment:
Missing test for LuaTableAdd, this only tests LuaTableAddMethod.

Reply:
I added more tests similar to existing lua table extensions tests.